### DISCUSSION TITLE
General Question

### DISCUSSION THREAD

**Discussion by itouchservices** [drakkan/sftpgo#1760] [Q&A]
Hello mates,
Thanks for helping in advance. I am hosting sftpgo using docker and I am wondering if the Admin user could see the folders created by the users created. I thought the Admin should be able to manage the folders and files created by the users.

**Comment by nzx-jsantaana** [comment]
In the SFTPGo model, "users" (who can access files) and "admins" (who can access the management interface) are entirely separate, with separate sets of credentials.  You can of course set up a user whose directory mappings let them see all the files you're serving, but that would still be a _user_ rather than an _admin_.

You could set up a user whose username was "admin", but they would still be separate to the default admin with that name.  (You should probably not do this, but only because it gets confusing for humans; there's no technical reason preventing it.)
